stubs/setuptools/pkg_resources/__init__.pyi

```diff
@@ -349,11 +427,12 @@ def safe_version(version: str) -> str: ...
 def safe_extra(extra: str) -> str: ...
 def to_filename(name: str) -> str: ...
 def get_build_platform() -> str: ...
-def get_platform() -> str: ...
+
+get_platform = get_build_platform
+
 def get_supported_platform() -> str: ...
 def compatible_platforms(provided: str | None, required: str | None) -> bool: ...
 def get_default_cache() -> str: ...
-def get_importer(path_item: str) -> _Importer: ...
```

Comment:
You also removed this on another PR, not sure why.

Otherwise this PR looks good.

Reply:
It's a re-import, (moved all the way to the top of the file).

Both PRs touched it because this change was relevant to both (it's now an import alias and it removes the _Importer class)

Reply:
Ah right, I missed that.